route53_crossfeed_app.tf

```
@@ -281,7 +281,7 @@ resource "aws_route53_record" "crossfeed_staging_MX" {
   zone_id = aws_route53_zone.cyber_dhs_gov.zone_id
 }
 
-resource "aws_route53_record" "crossfeed_staging_TXT" {
+resource "aws_route53_record" "crossfeed_staging_ses_TXT" {
```

See https://github.com/cisagov/cool-dns-cyber.dhs.gov/pull/80#discussion_r1288554962:
```suggestion
resource "aws_route53_record" "crossfeed_staging_TXT" {
```